os/binfmt/binfmt_exit.c

```diff
@@ -107,6 +111,19 @@ int binfmt_exit(FAR struct binary_s *bin)
 #endif
 	DEBUGASSERT(bin != NULL);
 
+#ifdef CONFIG_SAVE_BIN_SYMBOL
```

Comment:
I think it should be called when unload_module succeed. so you can move this to line 134.

Reply:
Done. Thanks.